EIPS/eip-1803.md

```diff
@@ -0,0 +1,36 @@
+---
+eip: 1803
+title: Rename opcodes for clarity
+author: Alex Beregszaszi (@axic)
+type: Standard Track
+category: Core
```

Comment:
This related to core logic, however doesn't relate to anything requiring a fork. I would suggest to change this to **Interface** as it is better described by 

> ... improvements around client API/RPC specifications and standards, and also certain language-level standards like method names

Reply:
My reasoning was that other EIPs refer to these names, therefore it is core. But I think I've solved that problem by #1968.

Also `SELFDESTRUCT` in https://eips.ethereum.org/EIPS/eip-6 is marked as `Interface` so changed it here too.